updates/6/updates.sql

```diff
@@ -233,4 +233,76 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xF962B098Ecc43
 *
 INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xFe8A3837cFf919C800bdC5d1ac6136F84497d679", 18, "yearn_vaults_v2"); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xFe8A3837cFf919C800bdC5d1ac6136F84497d679", "C", "UNI yVault", "yvUNI", 1614210195, NULL, "", "", "0xFe8A3837cFf919C800bdC5d1ac6136F84497d679"); INSERT INTO underlying_tokens_list(address, weight, parent_token_entry) VALUES ("0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984", "1.0", "0xFe8A3837cFf919C800bdC5d1ac6136F84497d679");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xeDF6568618A00C6F0908Bf7758A16F76B6E04aF9", 18, ""); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xeDF6568618A00C6F0908Bf7758A16F76B6E04aF9", "C", "ARIANEE", "ARIA20", 1558016715, NULL, "arianee", "ARIA20", "0xeDF6568618A00C6F0908Bf7758A16F76B6E04aF9");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("CRFI", "S", "Crossfi", "CRFI", 1620587551, NULL, "crossfi", "", "CRFI"); INSERT INTO common_asset_details(asset_id, forked) VALUES("CRFI", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xD417144312DbF50465b1C641d016962017Ef6240", 18, ""); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xD417144312DbF50465b1C641d016962017Ef6240", "C", "Covalent Query Token", "CQT", 1621556153, NULL, "covalent", "CQT", "0xD417144312DbF50465b1C641d016962017Ef6240");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("SLRS", "W", "Solrise Finance", "SLRS", 1625583749, NULL, "solrise-finance", "SLRS", "SLRS"); INSERT INTO common_asset_details(asset_id, forked) VALUES("SLRS", NULL);
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("SNY", "Y", "Synthetify Token", "SNY", 1625829480, NULL, "syntheify-token", "", "SNY"); INSERT INTO common_asset_details(asset_id, forked) VALUES("SNY", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xEA1ea0972fa092dd463f2968F9bB51Cc4c981D71", 18, ""); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xEA1ea0972fa092dd463f2968F9bB51Cc4c981D71", "C", "Modefi", "MOD", 1613369145, NULL, "modefi", "MOD", "0xEA1ea0972fa092dd463f2968F9bB51Cc4c981D71");
+*
```

Comment:
This token already exists in the DB. But seems to have no coingecko/cryptocompare. So you should just add the coingecko/cryptocompare instance.

```
sqlite> SELECT * from assets where identifier="_ceth_0xEA1ea0972fa092dd463f2968F9bB51Cc4c981D71";
_ceth_0xEA1ea0972fa092dd463f2968F9bB51Cc4c981D71|C|Modefi|MOD|||||0xEA1ea0972fa092dd463f2968F9bB51Cc4c981D71
sqlite> 
```

Reply:
As discussed in discord, I have it in my global DB but not in the one coming with v1.18.1. So that is a bit confusing. I don't remember adding this on my own manually at any point.

Reply:
As discussed in discord we are leaving this insert